src/notifications/__init__.py

```diff
@@ -130,6 +144,10 @@ def showEvent(self):
             modhtml = '' if (modstr == '') else '<br><font size="-4"><font color="red">mods</font> %s</font>' % modstr
             text = '<html>%s<br><font color="silver" size="-2">on</font> %s%s</html>' % \
                    (game.title, maps.getDisplayName(game.mapname), modhtml)
+        elif eventType == self.GAME_FULL:
+            player = data
+            pixmap = self.user
+            text = '<html><br><font color="silver" size="-2">Game is full ! press x and play !</font></html>'
```

Comment:
This string is shown to the user, please make sure it's presentable (flawless style / punctuation etc.)